data/examples/declaration/signature/specialize/specialize-out.hs

```diff
@@ -11,5 +11,9 @@ baz :: Num a => a -> a
 baz = id
 {-# SPECIALIZE [~2] baz
   :: Int
-  -> Int
+     -> Int
```

Comment:
I'm not sure why this happened. Suggestions?

Reply:
I think the interaction of `velt` and `withSep` will align `::` with the comma (whether they exist or not). Here is an example with multi-line with comma so you can see:

```haskell
       {-# SPECIALIZE fits13Bits
         :: Int -> Bool
         , Integer -> Bool
         #-}
```

I'm not sure there's much we can do about aligning with a single multi-line type (no comma). But when we have multi-line comma, maybe we could add an extra space after the comma?

Reply:
This happens because `velt` wraps every "item" is `sitcc`. This is OK.